BLSQ/iaso
# POLIO-944: Change '3-level call' field into 'Outbreak declaration date'

Explain what problem this PR is resolving
- Replace the 3 level call field with the Outbreak declaration date one
Related JIRA tickets : [POLIO-944](https://bluesquare.atlassian.net/browse/POLIO-944)

## Self proofreading checklist


## Changes
- Replace the '3 level call' by the 'outbreak declaration date' both in the base info and risk assessment tabs
- Update the translation
- Created db migration to replace three_level_call by outbreak_declaration_date

## How to test
- Go in campaign
- Create or edit a campaign
- Check in the base info and risk assessment tabs

## Print screen / video

![Screenshot from 2023-04-14 10-46-37](https://user-images.githubusercontent.com/19631540/231993516-1c83f98a-6306-4f24-a34f-ecd28f7c14d3.png)


![Screenshot from 2023-04-14 10-47-00](https://user-images.githubusercontent.com/19631540/231993674-930be126-cbbc-43bb-8fd9-3fd0f92b4f8e.png)



[POLIO-944]: https://bluesquare.atlassian.net/browse/POLIO-944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
